demos/using-dev-build/README.md

```diff
@@ -31,6 +41,25 @@ At this point, the replica (for all practical matters, a local version of the In
 
 If the IC actually lets the call (request) through to the `whoami` canister, it means that everything checked out, and the `whoami` canister just responds with the information the IC adds to requests, namely your identity (principal).
 
+### Adding Internet Identity to your Local Project
+
+This section explains how to add the [dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity to your (local) project. It is recommended to use the dev build locally because it has modifications that make test automation easy.
```

Comment:
I think mentioning the dev build here may confuse readers. How about not mentioning it at all in the main text but maybe add a note that users can skip? Most people will just not care, although I agree some will want to know more